outputs.tf

```diff
@@ -95,7 +95,7 @@ output "cluster_proportional_autoscaler" {
 
 output "fargate_fluentbit" {
   description = "Map of attributes of the Helm release and IRSA created"
-  value       = try(module.fargate_fluentbit[0], null)
+  value       = kubernetes_config_map_v1.aws_logging
```

Comment:
is this equivalent to the prior output?

Reply:
Not exactly the same, but it will return the configMap, configuration that should have basically the same information as the old module output. The old module also outputs the Namespace, but it is shown in the configMap as well.